### PR TITLE
Steve/140 symptoms admin

### PR DESCRIPTION
**Pull request by smarkandu**
Implementation for #140 (Symptoms in PatientProfile on Admin app)

### CODE REVIEW

File: Admin-app/src/components/PatientProfile/index.js [hunk 1]
```diff
@@ -137,6 +134,24 @@ function PatientProfile() {
...
+          statuses &&
+            setPatientInfoStatuses(
+              statuses.map((status) =>
+                createData(
+                  status?.timestamp?.toDate()?.toLocaleString() || "",
+                  status.fever || "No",
+                  status.cough || "No",
+                  status.runnyNose || "No",
+                  status.muscleAche || "No",
+                  status.tiredness || "No",
```
Comment (MarwaKhalid) [inline Admin-app/src/components/PatientProfile/index.js:147]: When the symptoms modal was created on the patient list, we replaced tiredness with sorethroat. So get the soreThroat field and change the table header to sore throat as well.

**Comment by MarwaKhalid** [comment]
Unsure if you meant to do this, but for patients that don't have symptoms in the backend, the list only has the header. Which kinda looks weird imo. Perhaps come up with a solution for that.

Another note, change the title of the table to "Status Details" and add 2 more columns for temperature & weight.

**Comment by MarwaKhalid** [comment]
Btw the relevant issue for this is: #43

File: Admin-app/src/components/PatientProfile/index.js (hunk 1 above)
Comment (smarkandu) [reply Admin-app/src/components/PatientProfile/index.js]: Corrected

**Comment by smarkandu** [comment]
> Unsure if you meant to do this, but for patients that don't have symptoms in the backend, the list only has the header. Which kinda looks weird imo. Perhaps come up with a solution for that.
> 
> Another note, change the title of the table to "Status Details" and add 2 more columns for temperature & weight. Remove the button next to symptoms details as well. We arent adding.

- Message added if table empty
- Title changed
- temperature and weight column added
- + button removed

**Review by MarwaKhalid — APPROVED** [review]
Reviewed and it's all good now!

**Pull request merged**